delft3dworker/admin.py

```diff
@@ -100,29 +102,37 @@ class Version_SVN_Admin(GuardedModelAdmin):
         SceneInline,
     ]
 
-
 @admin.register(GroupUsageSummary)
 class GroupUsageSummaryAdmin(admin.ModelAdmin):
     # Following example available at:
     # https://medium.com/@hakibenita/how-to-turn-django-admin-into-a-lightweight-dashboard-a0e0bbf609ad
 
     change_list_template = 'delft3dworker/group_summary_change_list.html'
-    # Sort by time period
-    date_hierarchy = 'user__scene__container__container_stoptime'
+    # Filter by time period
+    list_filter = (('user__scene__container__container_stoptime', DateRangeFilter),)
 
     def changelist_view(self, request, extra_context=None):
         """
         Displays summary of usage organized by group
         """
+        extra_context = extra_context or {}
+        try:
```

Comment:
It's nicer to use the get function (https://docs.python.org/2/library/stdtypes.html#dict.get) for a dict. It will provide a default field or returns None, instead of raising keyerrors. Also catching every exception is bad practice, define the error you're catching.